test/Sentry.Tests/Internals/SentryStackTraceFactoryTests.MethodGeneric_mode=Enhanced.verified.txt

```diff
@@ -2,6 +2,6 @@
   FileName: Internals/SentryStackTraceFactoryTests.cs,
   Function: void SentryStackTraceFactoryTests.GenericMethodThatThrows<T>(T value),
   AbsolutePath: {ProjectDirectory}Internals/SentryStackTraceFactoryTests.cs,
-  InApp: true,
```

Comment:
Where is this change coming from?

Reply:
The `parameter` being matched in this case is`Other.Tests.Internals.SentryStackTraceFactoryTests` (that's basically the namespace for our test). 

There's only one `InAppInclude`: `Sentry.Samples` so that doesn't match. 

And there is an exclude that matches: `Sentry` (which is a substring of our namespace). 

So, we deem it `InApp: false`.

Reply:
Actually, look at this with fresh eyes, I think I might have broken something. By changing from `string` to `SubstringOrRegexPattern` the matching logic inadvertedly got changed from `StartsWith` to `Substring`, which matches all kinds of excludes that it shouldn't. 

Fixing now...